### DISCUSSION TITLE
Mac防火墙例外已添加但仍然无法被访问

### DISCUSSION THREAD

**Discussion by xiamubobby** [AlistGo/alist#1273] [Q&A]
通过系统设置防火墙GUI和CLI命令：sudo /usr/libexec/ApplicationFirewall/socketfilterfw --unblockapp /opt/bin/alist-darwin-10.12-amd64均无法使局域网访问顺利。（同时localhost和127.0.0.1可以访问）

添加例外的就是下载下来的那个可执行文件。

但是关闭系统防火墙一切都没有问题了，是可以用的。（所以不是alist的部署有问题）
系统防火墙对于我日常使用的其他app和二进制都也没问题。（所以也不是防火墙本身有什么问题）

是我误解了让alist在mac上过墙的方法吗？
搜过论坛了，没有相关讨论。

<img width="562" alt="image" src="https://user-images.githubusercontent.com/810250/175798678-519b90e7-07f0-471c-8629-5f362d5adc6b.png">

```bash
 ~ % history | grep socketfilter |  grep -v history | tail -n 2
 1012  sudo /usr/libexec/ApplicationFirewall/socketfilterfw --add /opt/bin/alist-darwin-10.12-amd64
 1013  sudo /usr/libexec/ApplicationFirewall/socketfilterfw --unblockapp /opt/bin/alist-darwin-10.12-amd64
```

**Comment by yinyu985** [comment]
我+1，mac，防火墙直接关了，还是连不上。

**Comment by Mrered** [comment]
我也有这样的问题，同样试过关闭防火墙，关闭LuLu等各种能想到的情况，局域网访问我Mac上的Home Bridge管理页面，甚至使用IPV6在局域网之外也能访问，而Alist无论是局域网内其他设备，还是局域网之外的IPV6，或者使用Tailscale虚拟局域网都无法访问。
本来想弄个文件收集仓，但是现在只能用别的办法了。

**Comment by ds3szy** [comment]
为啥我有win挂载好了 mac无法访问，手机也不行……可能是局域网下的其他设备都不行

**Comment by chengnan049** [comment]
兄弟们莫急，等我换回黑苹果给大家测试下


刚刚在 Ventura 上进行测试，直接下载二进制文件进行赋权，无法连接，后放行端口仍然无法连接
从配置文件中更换 5244 端口为其他端口口其他设备均可通过 `WebDav` 方式连接到 Alist，遂将端口改回 5244 ，此时 5244 也可连接，从防火墙中删除规则也正常连接，日志无报错

---
Update: 5:25 PM Mar.26

**Comment by mochi-c** [comment]
有人解决了这个问题吗
